src/Workspaces/Core/Portable/CodeGeneration/CodeGenerationHelpers.cs

```diff
@@ -172,14 +172,10 @@ public static IEnumerable<ISymbol> GetMembers(INamedTypeSymbol namedType)
                             }).ToList();
         }
 
-        public static T? GetReuseableSyntaxNodeForSymbol<T>(ISymbol symbol, CodeGenerationOptions options)
-            where T : SyntaxNode
+        public static T RemoveLeadingDirectiveTrivia<T>(T node) where T : SyntaxNode
         {
-            Contract.ThrowIfNull(symbol);
-
-            return options != null && options.ReuseSyntax && symbol.DeclaringSyntaxReferences.Length == 1
-                ? symbol.DeclaringSyntaxReferences[0].GetSyntax() as T
-                : null;
+            var leadingTrivia = node.GetLeadingTrivia().Where(trivia => !trivia.IsDirective);
```

Comment:
did you add a test for:

```
// foo
#if true
// bar
```
?

Reply:
I guess you mean those two tests? https://github.com/Cosifne/roslyn/blob/cce30bd09655ab26767360b2f66d966b3a6a57af/src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs#L4020
And
https://github.com/Cosifne/roslyn/blob/cce30bd09655ab26767360b2f66d966b3a6a57af/src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs#L4051